index.js

```diff
@@ -1,9 +1,17 @@
 'use strict';
 var got = require('got');
-var registryUrl = require('registry-url');
+var npmKeywordUrl = require('npm-keyword-url');
```

Comment:
Its actually pretty useful. Im already using it in some internal work projects. At least until npm gives us a better api.